### DISCUSSION TITLE
How do I specify a preset on the vue create command line?

### DISCUSSION THREAD

**Discussion by spierepf** [vuejs/vue-cli#7111] [Q&A] [answered]
I am trying to run `npx @vue/cli create test` from python (using popen) in an effort to generate vue projects for automated testing.

What I would like to do is create a vue preset and then specify that preset on the command line.

How do I specify the vue preset on the vue cli command line?

**Comment by Viggo95** [comment, marked as answer]
like this

```
vue create --inlinePreset '{"plugins":{"@vue/cli-plugin-a":{}}}'  my-project
```
